.github/workflows/integration.yml

```diff
@@ -68,14 +68,46 @@ jobs:
           SNOWFLAKE_ROLE: ${{ secrets.SNOWFLAKE_ROLE }}
         run: make test-acceptance
 
-      - name: comment on pull request
-        if: github.event_name == 'repository_dispatch'
+      - name: find comment
+        if: ${{ always() }}
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Integration tests ran for
+
+      - name: create comment
+        if: (${{ failure() }} || ${{ cancelled() }} || ${{ success() }}) && ${{ steps.fc.outputs.comment-id == 0 }} && (github.event_name == 'repository_dispatch')
         uses: peter-evans/create-or-update-comment@v1
         with:
           issue-number: ${{ github.event.client_payload.pull_request.number }}
           body: |
             Integration tests ran for [${{ github.event.client_payload.slash_command.args.named.sha }}](https://github.com/chanzuckerberg/terraform-provider-snowflake/actions/runs/${{ github.run_id }})
-          reactions: '+1'
+
+      - name: create comment
+        if: (${{ failure() }} || ${{ cancelled() }} || ${{ success() }}) && ${{ steps.fc.outputs.comment-id == 0 }} && github.event_name == 'pull_request'
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.client_payload.pull_request.number }}
+          body: |
+            Integration tests ran for [${{ github.event.pull_request.head.sha }}](https://github.com/chanzuckerberg/terraform-provider-snowflake/actions/runs/${{ github.run_id }})
```

Comment:
ok to leave out the pul_request ones since they get linked automatically in the tests?

Reply:
ah yes you're right, I was thinking this but got distracted.